content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -446,6 +447,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `EphemeralContainers`: Enable the ability to add {{< glossary_tooltip text="ephemeral containers"
   term_id="ephemeral-container" >}} to running pods.
 - `EvenPodsSpread`: Enable pods to be scheduled evenly across topology domains. See [Pod Topology Spread Constraints](/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+- `ExecProbeTimeout`: Ensure kubelet respects exec probe timeouts. Feature gate exists in-case existing workloads may depend on old behavior where exec probe timeouts were ignored. See [readiness probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
```

Comment:
Is there a way to link directly to the docs for `timeoutSeconds` here?  Or the note section below talking about the behavior for timeoutSeconds?

Reply:
I recommend that https://k8s.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/ links to the API doc, if needed.

Reply:
@andrewsykim are you thinking of linking to this section directly `/#configure-probes`?

There is also a feature in modern browsers to link to text like this: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes:~:text=timeoutSeconds. But this site is not using this feature - my attempts were turned down.